src/test/java/webdriver/WebDriverSetup.java

```diff
@@ -20,7 +21,7 @@ public static WebDriver getDriver() {
 	}
 
 	private static void setupWebDriver() {
-		switch (System.getenv("WebDriver")) {
+		switch (new PropertyValue().getBrowser()) {
```

Comment:
Seems it's not so good solution - in any case when you need to get some properties - you need to create a new object of PropertyValue class. Let's do some googling and find a better way